<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#leftmenu{
	background-color: #67b8a5;

}

#frame_max, #frame_jitter, #frame_min, #frame_mean{
	background-color: #fefeff; 
	border-radius: 20px;
	border: #67b8a5;
 	box-shadow: 5px 5px 10px 5px #67b8a5;

}

#button_importfile{
	background-color: #67b8a5;
	color: #fff;
	border-radius: 10px;
}

#button_importfile_2{
	background-color: #67b8a5;
	color: #fff;
	border-radius: 10px;
}

#button_home{
	background-color: #fff;
	color: #302f2f;
	border-radius: 10px;
}


#button_latencyanalysis{
	background-color: #fff;
	color: #302f2f;
	border-radius: 10px;
}

#button_comparegraphics{
	background-color: #fff;
	color: #302f2f;
	border-radius: 10px;
}

#button_throughputanalysis{
	background-color: #fff;
	color: #302f2f;
	border-radius: 10px;
}

#button_addfile{
	background-color: #67b8a5;
	color: #fff;
	border-radius: 10px;
}


</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="leftmenu" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="leftmenu_title" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QLabel" name="label_q5ga">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>201</width>
            <height>111</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Frutiger LT Com 55 Roman</family>
            <pointsize>35</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 35pt &quot;Frutiger LT Com 55 Roman&quot;;
color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 255));
</string>
          </property>
          <property name="text">
           <string>Quick 5G 
Analyser</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="leftmenu_buttons" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>400</height>
          </size>
         </property>
         <widget class="QPushButton" name="button_latencyanalysis">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Latency analysis</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_brown/Brown Icons/activity.svg</normaloff>:/icons_brown/Brown Icons/activity.svg</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_throughputanalysis">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Throughput analysis</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_brown/Brown Icons/activity.svg</normaloff>:/icons_brown/Brown Icons/activity.svg</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_comparegraphics">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>300</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Compare graphics</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_brown/Brown Icons/bar-chart-2.svg</normaloff>:/icons_brown/Brown Icons/bar-chart-2.svg</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_home">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;</string>
          </property>
          <property name="text">
           <string>Home</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_brown/Brown Icons/home.svg</normaloff>:/icons_brown/Brown Icons/home.svg</iconset>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainbody" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="headerframe" native="true">
         <property name="minimumSize">
          <size>
           <width>680</width>
           <height>20</height>
          </size>
         </property>
         <widget class="QPushButton" name="button_importfile">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>10</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Frutiger LT Com 55 Roman</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Import Linux File</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_white/White Icons/arrow-down-circle.svg</normaloff>:/icons_white/White Icons/arrow-down-circle.svg</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_latency">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>120</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 25pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Latency</string>
          </property>
         </widget>
         <widget class="QPushButton" name="button_importfile_2">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>50</y>
            <width>161</width>
            <height>31</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Import Windows File</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_white/White Icons/arrow-down-circle.svg</normaloff>:/icons_white/White Icons/arrow-down-circle.svg</iconset>
          </property>
         </widget>
         <widget class="QLabel" name="label_latency_2">
          <property name="geometry">
           <rect>
            <x>550</x>
            <y>140</y>
            <width>61</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 15pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Values</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="middleframe" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>335</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>480</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QGraphicsView" name="graphicsView">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>471</width>
               <height>311</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QFrame" name="frame_max">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_max">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>0</y>
                  <width>81</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Maximum</string>
                </property>
               </widget>
               <widget class="QLabel" name="value_max">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>30</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_min">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_min">
                <property name="geometry">
                 <rect>
                  <x>40</x>
                  <y>0</y>
                  <width>81</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Minimum</string>
                </property>
               </widget>
               <widget class="QLabel" name="value_min">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>30</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_mean">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_mean">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>0</y>
                  <width>51</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Mean</string>
                </property>
               </widget>
               <widget class="QLabel" name="value_mean">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>30</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_jitter">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QLabel" name="label_jitter">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>0</y>
                  <width>61</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Jitter</string>
                </property>
               </widget>
               <widget class="QLabel" name="value_jitter">
                <property name="geometry">
                 <rect>
                  <x>60</x>
                  <y>30</y>
                  <width>41</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Frutiger LT Com 55 Roman</family>
                  <pointsize>24</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>30</string>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="bottomframe" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <widget class="QPushButton" name="button_addfile">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>20</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Frutiger LT Com 55 Roman</family>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;Frutiger LT Com 55 Roman&quot;;
</string>
          </property>
          <property name="text">
           <string>Add File</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons_white/White Icons/plus.svg</normaloff>:/icons_white/White Icons/plus.svg</iconset>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuick_5G_Analisys">
    <property name="title">
     <string>Quick 5G Analisys</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuQuick_5G_Analisys"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
